airbyte-integrations/connectors/source-file/requirements.txt

```diff
@@ -1,4 +1,6 @@
 -e ../../bases/airbyte-protocol
 -e ../../bases/base-python
 -e ../../bases/base-python-test
--e .
\ No newline at end of file
+-e .
+pytest
```

Comment:
should the go in setup.py instead?

Reply:
It is in there but my IDE still doesn't find it unless it is in the requirements.txt file...